src/components/App.js

```diff
@@ -18,7 +20,9 @@ export class App extends Component {
       <main>
         <NavigationBar />
 
-        <Route exact path='/' render={props => <MovieContainer {...props} />} />
+        <Route exact path='/' render={() => <MovieContainer />} />
+
+        <Route exact path='/login' render={() => <LoginForm />} />
```

Comment:
Since we aren't passing props here, would we need to use the `<Route render>` option or would `<Route component>` work?

`<Route render>`  example would be  `<Route path='/unicorns' render={ () => <Unicorns /> }`
`<Route component>` example would be `<Route path='/unicorns' component={ Unicorns } />`

Reply:
Good catch! I've amended the syntax and will push the changes up shortly